recipe/meta.yaml

```diff
@@ -27,20 +29,19 @@ requirements:
 test:
   requires:
     - python 2.7.*  # [win and py27]
-    - python 3.4.*  # [win and py34]
     - python 3.5.*  # [win and py35]
```

Comment:
Maybe we should just simplify this to one line (as below) now that `environ` is better supported in `conda-forge` and `conda-build`.

```yaml
    - python {{ environ['PY_VER'] + '*' }}  # [win]
```